microcosm_flask/conventions/health.py

```diff
@@ -47,6 +51,9 @@ def to_dict(self):
         )
 
 
+@defaults(
+    path_prefix="/api",
```

Comment:
I like the `/api` convention because it leaves room for serving `/admin` or static content.